alchemiscale/compute/api.py

```diff
@@ -76,6 +80,17 @@ async def check(
     _check_store_connectivity(n4js, s3os)
 
 
+@router.get("/identities/{identity_identifier}/scopes")
```

Comment:
The typical pattern for RESTful APIs is to go `/<entity_plural>/<entity_identifier>/`, with anything following being used to grab something specific for that entity, such as `scopes` like we do here. We use this pattern on other endpoints for this API as well, so I've made this change here to reflect that.